src/NServiceBus.Persistence.DynamoDB.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -70,6 +70,12 @@ namespace NServiceBus.Persistence.DynamoDB
         public static object? ToObject(System.Collections.Generic.Dictionary<string, Amazon.DynamoDBv2.Model.AttributeValue> attributeValues, System.Type returnType) { }
         public static TValue? ToObject<TValue>(System.Collections.Generic.Dictionary<string, Amazon.DynamoDBv2.Model.AttributeValue> attributeValues) { }
     }
+    [System.Serializable]
+    public class PartialOutboxResultException : System.Exception
```

Comment:
Wasn't really sure what name to pick so I went with whatever came to my mind first. Maybe you have some good suggestions?

Reply:
I think it's ok. Alternatives might be `PartialQueryResultException`, `InconsistentQueryResultException`, `MissingOutboxRecordsException` but none of them feel clearly superior to me.